queenbee/cli/repository/manage.py

```diff
@@ -0,0 +1,70 @@
+import os
+import json
+from urllib.parse import urlparse
+
+try:
+    import click
+    from click_plugins import with_plugins
+except ImportError:
+    raise ImportError(
+        'click modules not installed. Try `pip install queenbee[cli]` command.'
+    )
+
+from ...config.repositories import RepositoryReference
+from ...repository.index import RepositoryIndex, RepositoryMetadata
+
+@click.command('add')
+@click.argument('name')
+@click.argument('path')
+@click.option('-f', '--force', help='Boolean toggle to overwrite existing repository with same name', default=False, type=bool, is_flag=True)
```

Comment:
PEP8 👮 says "The speed limit is 89 characters per line."

Reply:
Yep... I'm counting on running a big `autopep8` lint on this. Was wanting to let you guys read through it first so the linting changes don't make the PR changes unmanageable 🙃